tools/online_editor/src/proxy.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import {
+  MessageReader,
+  MessageWriter,
+  DataCallback,
+  Message,
+  Disposable,
+  Event,
+  PartialMessageInfo,
+} from "monaco-languageclient";
+
+export class ProxyReader implements MessageReader {
```

Comment:
Naming wise I suggest to include "Filter" in the name, because it's not just a proxy - it allows filtering messages entirely. "FilterProxyReader" perhaps.

Reply:
Good point, done.